xapian-applications/omega/omega.cc

```diff
@@ -447,6 +479,11 @@ try {
     // Percentage relevance cut-off
     val = cgi_params.find("THRESHOLD");
     if (val != cgi_params.end()) {
+	if (val->second[0] < '0' || val->second[0] > '9' ||
+	    val->second[0] == '-') {
+	    cerr << "Value entered is not in range\n"
+		<< "Range is 0-100";
+	}
```

Comment:
The check isn't what the error message implies - e.g. `101` is quietly accepted.

Reply:
thanks i will fix that.